tests/Prometheus.Client.HttpClient.Tests/Prometheus.Client.HttpClient.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>E:\Dev\Repos\OSS\prom-client-httpclient\Prometheus.Client.HttpClient.snk</AssemblyOriginatorKeyFile>
```

Comment:
Link to snk already in `Directory.Build.props`

https://github.com/prom-client-net/prom-client-httpclient/blob/74523035fb587ba514ff6cae0efc3ceddf484cd5/Directory.Build.props#L19